shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-distsql/shardingsphere-shadow-distsql-parser/src/main/antlr4/imports/shadow/Literals.g4

```diff
@@ -25,7 +25,7 @@ STRING
     ;
 
 IDENTIFIER
-    : [A-Za-z_$0-9]*?[A-Za-z_$]+?[A-Za-z_$0-9]*
+    : [A-Za-z_$0-9-]*?[A-Za-z_$-]+?[A-Za-z_$0-9-]*
     | BQ ~'`'+ BQ
```

Comment:
In order to identify the name with a'-', such as ‘user-id-simple-note-algorithms’.